examples/tutorials/data/hawc.py

```diff
@@ -0,0 +1,257 @@
+obs.bkg.reduce_over_axes().plot(add_cbar=True)
+
+######################################################################
+# Peek the effective exposure for one transit
+plt.figure()
+obs.aeff.reduce_over_axes().plot(add_cbar=True)
+
+
+######################################################################
+# Data reduction into a MapDataset
+# --------------------------------
+#
+# We will now produce a MapDataset using the data from one of the
+# fHit bins. In order to use all bins, one just needs to repeat this
+# process inside of a for loop that modifies the variable fHit.
+
+
+######################################################################
+# First we define the geometry and axes
+
+# create the energy reco axis
+energy_axis = MapAxis.from_edges([1.00,1.78,3.16,5.62,10.0,17.8,31.6,56.2,100,177,316] * u.TeV,
+        name="energy",
+        interp="log")
+
+
+# and energy true axis
+energy_axis_true = MapAxis.from_energy_bounds(1e-3, 1e4, nbin=140, unit="TeV", name="energy_true")
+
+
+# create a geometry around the Crab location
+geom = WcsGeom.create(skydir=SkyCoord(ra=83.63,dec=22.01, unit='deg', frame="icrs"),
+                     width=6*u.deg,
+                     axes=[energy_axis],
+                     binsz=0.05)
+
+
+######################################################################
+# Define the makers we will use
+
+maker = MapDatasetMaker(selection = ["counts", "background", "exposure", "edisp", "psf"])
+safemask_maker = SafeMaskMaker(methods=['aeff-max'], aeff_percent=10)
+
+
+######################################################################
+# Create empty Mapdataset
+dataset_empty = MapDataset.create(geom,
+                                  energy_axis_true= energy_axis_true,
+                                  name ='fHit ' + str(fHit),
+                                  reco_psf=True )
+# run the map dataset maker
+dataset = maker.run(dataset_empty, obs)
+
+# The livetime info is used by the SafeMask maker to retrieve the
+# effective area from the exposure. The HAWC effective area is computed
+# for one source transit above 45º zenith, which is around 6h
+# Note that since the effective area condition used here is relative to
+# the maximum, this value does not actually impact the result
+dataset.exposure.meta["livetime"] = "6 h"
+dataset = safemask_maker.run(dataset)
+
+
+######################################################################
+# Now we have a dataset that has background and exposure quantities for
+# one single transit, but our dataset might comprise more. The number
+# of transits can be derived using the good time intervals (GTI) stored
+# with the event list. For convenience, the HAWC data release already
+# included this quantity as a map
+
+transit_map = Map.read(data_path + 'irfs/TransitsMap_Crab.fits.gz')
+transit_number = transit_map.get_by_coord(geom.center_skydir)
+
+# Correct the quantities
+dataset.background.data*=transit_number
+dataset.exposure.data*=transit_number
```

Comment:
could this be done automatically by the MapDatasetMaker ?

Reply:
see comment above - yes it could and would be nice, but would also mean a significant new feature (i.e. computing exposure from GTIs)